config/_default/menus/menus.en.yaml

```diff
@@ -1526,11 +1526,16 @@ main:
     parent: tracing_custom_inst
     identifier: tracing_custom_inst_dotnet
     weight: 108
-  - name: Instrumenting with OpenStandards
-    url: opentelemetry/otel_tracing/
+  - name: OpenStandards
+    url: tracing/trace_collection/open_standards
     parent: tracing_trace_collection
     identifier: tracing_open_standards
-    weight: 105
+    weight: 105  
```

Comment:
I'm not sure we need both this page (which should have a space between "Open" and "Standards") and the Otel page below it. Maybe if we added the language next-links section that this page has into the Otel page below it, and just get rid of this page?

Reply:
I definitely overthought this. I changed the Nav title to OpenTelemetry and added the next-links like you recommended!